packages/components/src/components/Tabs/Tabs.js

```diff
@@ -19,7 +19,7 @@ import {
 } from 'carbon-components-react';
 
 const TabContent = ({ selected, ...other }) =>
-  selected && <CarbonTabContent selected={selected} {...other} />;
+  selected ? <CarbonTabContent selected={selected} {...other} /> : <span></span>;
```

Comment:
Shouldn't this pass the id to the span? We'll also need to include styles to ensure it's visibly hidden as we don't want unexpected content (even if it's just whitespace) appearing on the page and potentially affecting layout etc.